README.md

```diff
@@ -1,6 +1,6 @@
 [![Maintained by Gruntwork.io](https://img.shields.io/badge/maintained%20by-gruntwork.io-%235849a6.svg)](https://gruntwork.io/?ref=repo_package_terraform_utilities)
 [![GitHub tag (latest SemVer)](https://img.shields.io/github/tag/gruntwork-io/package-terraform-utilities.svg?label=latest)](https://github.com/gruntwork-io/package-terraform-utilities/releases/latest)
-![Terraform Version](https://img.shields.io/badge/tf-%3E%3D0.12.0-blue.svg)
+![Terraform Version](https://img.shields.io/badge/tf-%3E%3D0.13.0-blue.svg)
```

Comment:
@brikis98 @infraredgirl @ina-stoyanova @marinalimeira should this be 0.14? Or was it 0.13 intentional?

Reply:
FTR, I followed [this TODO](https://github.com/gruntwork-io/prototypes/pull/110/commits/8b455ac5db5b48c4993818d500030a352303d0da#diff-3f9c9012ec26ca089c1280870bbdc28bafc09af209e02a5e3f66a4eaf74e7e79L301) that said `0.13`. However, thinking more about it, I'm not sure `0.13` makes sense. Our code is still pinned to `0.12.26` (the script makes not changes there), and the current badge says `0.12`. Should we just leave it at `0.12`?

Reply:
I'd set it to 0.14, as that's what we're testing with now.

Reply:
Or, alternatively, we could try to create a badge, or two badges, that shows we support 0.12-0.14. But going to solely 0.14 is prob simpler and more accurate given the testing strategy.

Reply:
Done in e7853ca.